tensorflow_lite_support/ios/task/processor/sources/TFLClassificationResult.m

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#import "tensorflow_lite_support/ios/task/processor/sources/TFLClassificationResult.h"
+
+@implementation TFLCategory
+@synthesize displayName;
+@synthesize label;
+@synthesize score;
+@synthesize classIndex;
+@end
+
+@implementation TFLClassifications
+@synthesize headIndex;
+@synthesize categories;
+
+- (instancetype)initWithCategories:(NSArray<TFLCategory *> *)categories headIndex:(int)headIndex {
```

Comment:
`initWithCategories` may need to update if there's a new property added to `TFLClassifications`, and then it will break existing clients.

Should we just reply on set/get methods generated through synthesize? Same for TFLClassificationResult.

Reply:
Do you mean to say not have a custom init and just assign the properties from the class in which it is initialised?

Reply:
Yes that's correct.

Reply:
Allright. I have fixed it.